.gitattributes

```diff
@@ -6,4 +6,5 @@ core.eol=lf
 *.qm binary
 *.zip binary
 
+dist/windows/license.txt text eol=crlf
```

Comment:
Not sure if you have notice it. To pass file health CI check you need to add the new license file to:
https://github.com/qbittorrent/qBittorrent/blob/8d5022c715ca2646cedf42fd59c24bdbe86f07a2/.pre-commit-config.yaml#L37

and remove a superfluous line at the end of ci_windows.yaml file.